springdb/data

```diff
@@ -0,0 +1 @@
+{[{"dataset":"Targets", "description":"Core annotation for targets", "source":"ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/latest/output/etl/parquet/targets", "timestamp":1647831895},{"dataset":"Drugs", "description":"Core annotation for drugs", "source":"ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/latest/output/etl/parquet/molecule", timestamp:1647831895},{"dataset":"Adverse Events", "description":"Significant adverse events for drug molecules", "source":"ftp://ftp.ebi.ac.uk/pub/databases/opentargets/platform/latest/output/etl/parquet/fda/significantAdverseDrugReactions", "timestamp":1647831895}]}
```

Comment:
I'm not sure why this file is here.  Should it be deleted?

Reply:
Yes, it should be deleted. It was for testing purposes.